src/shared/Git-Credential-Manager/Program.cs

```diff
@@ -86,7 +89,7 @@ public static void Main(string[] args)
                     .GetAwaiter()
                     .GetResult();
 
-                context.Trace2.Stop(exitCode);
+                context.Trace2.Stop(exitCode, Thread.CurrentThread.Name);
```

Comment:
Every call to `Stop` takes `Thread.CurrentThread.Name`.. perhaps we should also use `BuildThreadName`?

Reply:
That is incorrect and should be removed - thanks for catching!